Lottie.sln

```diff
@@ -707,7 +707,6 @@ Global
 		{A7CECB0B-C674-462C-8E50-6A21BC41A62D}.Debug|x86.ActiveCfg = Debug|Any CPU
 		{A7CECB0B-C674-462C-8E50-6A21BC41A62D}.Debug|x86.Build.0 = Debug|Any CPU
 		{A7CECB0B-C674-462C-8E50-6A21BC41A62D}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{A7CECB0B-C674-462C-8E50-6A21BC41A62D}.Release|Any CPU.Build.0 = Release|Any CPU
```

Comment:
It looks like the release configuration was trying to build the Forms macOS project, while all other targets were removed. Is there are reason for it being the sample only project built?